UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/Program.cs

```diff
@@ -38,8 +40,8 @@ public static async Task Main()
                 ServiceProvider serviceProvider = serviceCollection.BuildServiceProvider();
                 try
                 {
-                    var bessConfigurationServiceJob = serviceProvider.GetService<BESSConfigurationServiceJob>();
-                    await bessConfigurationServiceJob.CreateBespokeExchangeSetAsync();
+                    var bessConfigurationServiceJob = serviceProvider.GetService<BessConfigurationServiceJob>();
+                    bessConfigurationServiceJob.Start();
```

Comment:
Any reason we have changed the method name to start here? 'CreateBespokeExchangeSetAsync' is more descriptive in my opinion